_devconfig/index.md

```diff
@@ -19,4 +17,4 @@ Future developer guides may cover the following topics:
 * How do I enable a domain to assert assurance in AD?
 * How do I validate trust stores on a Windows platform?
 
-To help contribute to this effort, please follow 'Submit Issues Here' link at the top right to access the GitHub page for this site and provide any feedback.
\ No newline at end of file
+You can [contribute]({{ site.baseurl }}/contribute/) to this effort, by providing feedback through [GitHub issues]({{site.github.repository_url}}/issues).
```

Comment:
I reworded this to not make people go up to the top right and figure out new context.